index.html

```diff
@@ -5,17 +5,57 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta name="description" content="This is the project that demonstrates the learners understanding of documenting professionally as well as using linters">
-    <title>Hello Microverse</title>
+    <meta name="description"
+        content="This is the description of william' portfolio">
+    <title>William | Portfolio</title>
     <link rel="shortcut icon" href="favicon.ico" type="image/x-icon">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap"
+        rel="stylesheet">
 </head>
 
 <body>
     <link rel="preload" href="styles.css" as="style" onload="this.onload=null;this.rel='stylesheet'">
     <noscript>
         <link rel="stylesheet" href="styles.css">
     </noscript>
-    <h1 class="header">Hello Microverse!</h1>
+    <div class="wrap">
+        <header class="toolbar">
+            <h4>My Logo</h4><img src="assets/img/bugger.png" alt="" />
```

Comment:
Hey 👋🏽  great design 🚀  but let's see how we can improve it:
- [ ] The logo could be on the left and have a link (a tag).
    - This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.
- [ ] The hamburger button could be on the right, and have a nav tag.
    - The nav tag is for semantics reason, and the right position for convention and user experience for mobile devices.

Reply:
Hey @didierganthier I have implemented the feedback and added this screenshot for context

![screens](https://user-images.githubusercontent.com/65118995/212466915-8835a658-f0a0-48db-a47d-7aa306fbb694.png)